website/docs/cdktf/index.html.md

```diff
@@ -30,9 +29,13 @@ You can use every Terraform provider and module available on the [Terraform Regi
 
 ## When to use CDK for Terraform
 
-CDK for Terraform should be used when users have a strong preference for a procedural language, or when the ability to create abstractions using Constructs is needed to help manage complexity. Users who are already familiar with HCL and do not need these capabilities should continue using HCL Terraform. This choice can be made team-by-team and project-by-project, because CDK for Terraform [interoperates with existing Terraform providers and modules](./concepts/interoperability-workflows.html)
+CDKTF offers many benefits, but it is not the right choice for every project. You should consider using CDKTF when:
 
-**TODO:** Insert content about when folks should use this instead of HCL... link to HCL interoperability page.
+-  You have a strong preference or need to use a procedural language to define infrastructure.
+- You need to create abstractions to help manage complexity. TO DO - example here
```

Comment:
maybe link this to the constructs documentation, and then make sure we cover "why" as well as "how" in there?

Reply:
yes! I'll add a link but..can you please just finish this sentence: "For example, you want to create constructs (link) to ......" I just think this is the time to emphasize the power of CDKTF to get people interested and show how it can solve problems. And so what's something someone might really be like OH WOW I want to make a construct for that!

Reply:
`for example, you want to create constructs to model a reusable infrastructure pattern composed of multiple resources and convenience methods.`

Reply:
perfect! I pushed this up :-)